tests/test_dse.py

```diff
@@ -101,28 +101,6 @@ def tti_nodse():
     return v, rec
 
 
-@skipif_yask
```

Comment:
Has this test been replaced by something else?

Reply:
That test is simply unmaintainable with the changes shipped by this branch. However, it tests a subsets of the things already tested by the other dse/tti tests, so we're not decreasing coverage.